commons/commands.py

```diff
@@ -153,6 +153,27 @@
 IPMI_SDR_TYPE_CMD = "ipmitool sdr type"
 IPMI_EVENT_CMD = "ipmitool event"
 
+#Provisioner AWS Commands
```

Comment:
These commands are not required , used existing method to fulfil your test requirement.
use post_deployment_lc method it will configure the client ,aws, access,secret key, even IO operations method are also exists.
all these s3 operations code already exists why to write new commands.